patch/driver-arista-restrict-eMMC-drive-to-50Mhz-from-userland.patch

```diff
@@ -0,0 +1,70 @@
+mmc: restrict eMMC drive to 50Mhz from userland
+
+This issue was fixed for kernel 3.18 by setting sdhci.debug_quirks2=0x40
+from Aboot boot0 (conditionally, for specific Aboot versions).
```

Comment:
What is Aboot?

Reply:
Aboot is the bootloader that is used on Arista switches (instead of GRUB). All of the other switches supported by SONiC use GRUB.

Reply:
That's right, there's also `u-boot` support in SONiC.
But independently of the bootloader this just means adding the parameter to the kernel cmdline.